### PR TITLE
Improve the example Babel config in the readme

### PR DESCRIPTION
**Pull request by grantglidewell**
The example babel config is incomplete. This works, could save some other people time.

### CODE REVIEW

**Comment by sindresorhus** [comment]
How exactly is it incorrect? The point of it is to transpile JSX, not to transpile unsupported JS features in Node.js.

**Comment by grantglidewell** [comment]
Sorry, incomplete would be more accurate. Using the information in the
readme, I was unable to get anything running on the command line. Node is
the target and this configurations works.

If there is something else I’m missing you might consider adding it to the
readme so other users are able to get this running. I’m just trying to be
helpful and get people who may be less familiar with Babel up and going
faster.

**Comment by grantglidewell** [comment]
My initial description was unclear, apologies. to describe the issue more precisely, if I follow the readme directly it requires me to research how I want to use babel in order to get up and going with Ink. With this change to the documentation, users can immediately start using Ink without having to do that extra step.

**Comment by vadimdemedes** [comment]
What syntax exactly caused an error for you?

**Comment by grantglidewell** [comment]
import syntax is technically whats causing the error (the babel config in the readme doesnt handle it).

File: readme.md [hunk 1]
```diff
@@ -95,13 +95,19 @@ The key difference you have to remember is that the rendering result isn't a DOM
...
-		]
-	}
-}
+  "babel": {
+    "presets": [
+      "@babel/preset-react",
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
```
Comment (vadimdemedes) [inline readme.md]: Is it possible to skip `targets` to make the config shorter? Also, could you use tabs here, not spaces?

Comment (grantglidewell) [reply readme.md]: if you want something that works, in node, this is the most reliable way to do it.
https://babeljs.io/docs/en/babel-preset-env#targetsnode
happy to add this
https://prettier.io/docs/en/options.html#tabs

Comment (vadimdemedes) [reply readme.md]: Nope, no need to add prettier, you can just convert spaces to tabs by hand.

**Pull request merged**